resources/api/src/main/java/org/jboss/forge/addon/resource/DefaultFileOperations.java

```diff
@@ -145,10 +150,23 @@ public void copyFile(File srcFile, File destFile) throws IOException
       doCopyFile(srcFile, destFile);
    }
 
+   @Override
+   public File move(File source, File target) throws IOException
+   {
+      if (target.isDirectory())
+      {
+         Path path = Files.move(source.toPath(), Paths.get(target.getAbsolutePath(), source.getName()),
```

Comment:
What happens if only Line 162 is executed? Eg: source as a file and target as a directory?